### ISSUE TITLE
Can't input <space> twice quickly

### ISSUE THREAD

**Issue by jonasmerlin** [VSCodeVim/Vim#6508]
**Describe the bug**
I mapped my <leader> key to <space> and enabled the EasyMotion plugin. So the command to activate EasyMotion would be <space><space>. But even though I can hit <space> twice in quick succession basically anywhere (including here), when I try to do it in VSCode it doesn't register the second <space> if I do it reasonably quick.

**Expected behavior**
My expected behavior would be that I can hit <space> as fast as anywhere else. 

**Environment:**

- Extension (VsCodeVim) version: 1.20.0
- VSCode version: 1.55.2
- OS: macOS

**Comment by jonasmerlin** [comment]
Update: by now I found out that thir only happens sometimes, when I press another key before the <space>. hjkl for example, so it seems that there is something that gets in the way of each other. Any idea what that could be?

**Comment by MarcoBuess** [comment]
I'm experiencing a similar issue which is occuring since I started using `vim.normalModeKeyBindingsNonRecursive` that involve the `<leader>` key. I also have my `<leader>` bound to `<space>`. Here is a sample keybinding I'm using.

```json
    "vim.normalModeKeyBindingsNonRecursive"           : [
        {
            "before"  : ["<leader>", "e"],
            "commands": ["workbench.view.explorer"]
        }
```

My guess is that, as I don't have a `"<leader> <leader>", "<x>"` custom keybinding, it isn't able to recognize `<leader><leader>`. Possible workaround would be a vim command to trigger `easymotion` if there is any.

**Comment by jonasmerlin** [comment]
There is another one. Finally 🥺

I just checked: I'm not using any of the non recursive bindings, so it might still be somethin else for you as well?

**Comment by MarcoBuess** [comment]
There must have been recent changes. The issue is gone for me. Wishing you the best of luck @jonasmerlin

**Comment by J-Fields** [comment]
@jonasmerlin Does the above solve it for you as well?

**Comment by Marskey** [comment]
yeah, I think it should be put into the docs. I was confused by it until I found this issue

**Comment by garrofederico** [comment]
Thanks @jonasmerlin for posting this issue and thanks @philipd for posting the solution!
You guys saved me a lot of time!! :D

**Comment by waltertondu** [comment]
Two years later.  This fix works to cure spacebar repeats in vscode->vim.

**Comment by simodoe** [comment]
yep I'm here in 2025 thinking the issue had to do with IdeaVim and/or PyCharm. D'oh. (I just went through and disabled all kinds of annoying "smart" Mac settings, so it worked out!)

thanks!
